src/en/users/getting-started/index.html

```diff
@@ -34,39 +34,25 @@ <h2 class="h2 mb-12 lg:mb-16" id="get-ceph">Get Ceph</h2>
         The quickest way to get a Ceph cluster up and running is to follow the Getting Started guides.
       </p>
       <p class="p">
-        These guides will show you how to build a working clusted, deploy gateways, and configure clients.
+        These guides will show you how to build a working cluster, deploy gateways, and configure clients.
       </p>
-      <a class="button" href="/{{ locale }}/users/documentation/">Getting started guide</a>
+      <a class="button" href="https://docs.ceph.com/en/latest/start/intro/">Getting started guide</a>
     </div>
     <div class="bg-grey-300 p-5 rounded-2">
       <h3 class="h3">Getting Ceph</h3>
       <p class="mb-8 lg:mb-12 p">
-        Pellentesque ornare sem lacinia quam venenatis vestibulum stibulum id ligula porta euismod semper.
+      Ceph offers clones of the Ceph project from Github as well as tarballs.
       </p>
       <div class="flex flex--gap-4 lg:flex--gap-6 flex--justify-between flex--wrap mb-8 lg:mb-12">
         <div class="relative text-center">
           <img alt="" class="block mb-2 mx-auto w-12" src="/assets/svgs/icon-git-blue-red.svg" />
-          <a class="a link-cover" href="/{{ locale }}/users/getting-started/">GIT</a>
+          <a class="a link-cover" href="https://github.com/ceph/ceph"/>GIT</a>
```

Comment:
Before git and tarball, can we have a ``Packages`` link to ``https://docs.ceph.com/en/latest/install/get-packages/`` and ``Containers`` link to ``https://docs.ceph.com/en/latest/install/containers/``

Reply:
Done.